data/minio/bucket/custom_files/training_params.yaml

```diff
@@ -20,6 +20,5 @@ METRIC_NAME: "TrainingRewardScore"
 CAR_COLOR: "Purple"
 TARGET_REWARD_SCORE: "None"
 NUMBER_OF_OBSTACLES: "3"
-CHANGE_START_POSITION: "true"
```

Comment:
Why was this param removed?

Reply:
It was duplicated inside the file.
I should have marked this change.

Reply:
The param is still there on line 5